### DISCUSSION TITLE
Expose port of podman node

### DISCUSSION THREAD

**Discussion by malenaluri** [coreemu/core#864] [Q&A] [answered]
Hi everyone,
I have a scenario involving two podman nodes: one is a database server and the other is attempting to connect to it. To establish a connection, the port of the podman server node must be exposed. However, the podman documentation only explains how to expose a port when creating a container. 
The emulator executes the podman run command without exposing any of the ports. I attempted to create an image of the running container initialized by core, and then recreate and start a container from that image to expose the required port. However, this workaround was unsuccessful.

Any ideas or comments?

**Comment by bharnden** [comment, marked as answer]
The way CORE sets up the containers exposing ports is not something you need to do, so not sure what is going on in your case.
